spec/features/admin/shipping_methods_spec.rb

```diff
@@ -23,12 +23,14 @@
       page.should have_field 'shipping_method_name'
       page.should have_field 'shipping_method_description'
       page.should have_select 'shipping_method_display_on'
-      page.should have_field 'shipping_method_require_ship_address_true', checked: true
+      page.should have_css 'div#shipping_method_zones_field'
 
+      # binding.pry
       # When I create a shipping method and set the distributors
       fill_in 'shipping_method_name', with: 'Carrier Pidgeon'
       check "shipping_method_distributor_ids_#{d1.id}"
       check "shipping_method_distributor_ids_#{d2.id}"
+      check "shipping_method_shipping_categories_"
```

Comment:
why the underscore? I am just asking, I dont know what it should be.

Reply:
That's how the selector appears in browser, it seems strange to me as well but rspec finds it and checks it, so all green here :/

Reply:
I understand that. But why is it like that in the page? can you check the page source code and see what is creating that selector without an id?

Reply:
It's coming form `check_box_tag`, I'm not sure why though but I keep investigating

Reply:
It is sent to the server all right, I think he empty [] was just put there because Spree doesn't use the name of this particular field so they don't care. We can either put something like `shipping_method[shipping_categories][#{category.id)]` to have the correct id in the name (I tested it and it works fine), or we can just leave it like that. Since it's our view now I believe it's no harm to change it as we want. What do you guys think ?

Reply:
great, thanks for investigating! I think you can decide. I'd put the category id in there.